debian8/templates/csv/mounts.csv

```diff
@@ -1,4 +1,10 @@
 # mountpoint options that should be set
```

Comment:
It would be good make it clear that when there is no mount point option defined it is just about mount points in separate partition. Not that we are checking that the mount point option has no options.

Would be good to have it here as a comment, but also somewhere as documentation.

Reply:
I will look into that, good point.

Reply:
Updated comments, hope it's ok now